test/js/bun/test/snapshot-tests/snapshots/__snapshots__/snapshot.test.ts.snap

```diff
@@ -368,3 +368,11 @@ exports[`most types: testing 7 2`] = `9`;
 exports[`most types: testing 7 3`] = `8`;
 
 exports[`most types: undefined 1`] = `undefined`;
+
+exports[`should work with expect.anything() 1`] = `
```

Comment:
oh if we change these to be UTF16 strings, i think it will properly escape them including handling other cases like unicode escape sequences and octal literals and the like

Reply:
Cool, I'll give that a shot